bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -112,6 +113,16 @@ public final class Image extends Resource implements Drawable {
 	 */
 	GC memGC;
 
+	/**
+	 * Base image data at zoom factor stored in baseDataZoomFactor
+	 */
+	private ImageData dataAtBaseZoomFactor;
+
+	/**
+	 * Zoom level for image data stored in <em>dataAtBaseZoomFactor</em>
+	 */
+	private int dataBaseZoomFactor;
```

Comment:
Since these values belong together, they be encapsulated into an object/record. There is already the `ElementAtZoom` record in `DPIUtil` for this purpose. Maybe that can be reused and, e.g., moved to some more general place than the `DPIUtil` class, since that record combining some element with a zoom level seems to be of general interest in SWT.

Reply:
I would like to postpone that to our PR when we refactored the Image to work properly with multiple zoom levels in parallel.

Reply:
Might be okay, but just to properly understand: is there a specific reason to postpone it and not do it now?